copy-framework.sh

```diff
@@ -0,0 +1,16 @@
+# MobileDevice.framework can live in two locations, the first is blocked from being accessed when
+# building so we need to copy it to our build directory to ensure that we can use it during the
+# build process. The second location is seemingly for updated versions of the framework and the
+# first location will actually become a symlink to this destination if it exists.
+
+MOBILEDEVICE_LOCATION=$1
+MOBILEDEVICE_NEW_LOCATION=$2
+BUILD_DIRECTORY=$3
+
+if [ -d $MOBILEDEVICE_LOCATION ]; then
+    cp -RL $MOBILEDEVICE_LOCATION $BUILD_DIRECTORY
+fi
```

Comment:
I don't think an else would work here, as it's most likely both locations will exist. I think the only way to avoid doing 2 copies is change the if statements to not copy the directory but to just change what we're copying

Reply:
Pushed a change that just changes what we're going to copy rather than ultimately copying twice